.github/ISSUE_TEMPLATE/---report-a-bug.md

```diff
@@ -26,13 +26,15 @@ If you can, add screenshots to show the problem. If you need to show a code exam
 ## 🖥 Desktop
  - OS: [e.g. macOS]
  - Browser: [e.g. chrome, safari]
- - Version: [e.g. 22]
+ - Browser Version: [e.g. 22]
+- ICDS Website Version [V1, V2]
```

Comment:
```suggestion
- ICDS Website Version [e.g. V1, V2]
```
added "e.g." to be consistent with other details. 

Just a question though, does "V1" mean the "old" v1 site? Just wondering if we say something like "As displayed on the site top navigation" ?

Reply:
Yes will add that in too.